test/e2e/components/cookie.js

```diff
@@ -0,0 +1,47 @@
+class Cookies {
+    constructor(driver) {
+        this._driver = driver;
+    }
+
+    async get(name) {
+        const cookies = await this._driver.manage().getCookies();
+        return new Cookie(cookies.find(cookie => {
+            return cookie.name === name;
+        }));
+    }
+
+    async getCookieConsentCookie() {
+        return await this.get('vl-cookie-consent-cookie-consent');
+    }
+
+    async getCookieConsentDateCookie() {
+        return await this.get('vl-cookie-consent-cookie-consent-date');
```

Comment:
zijn de namen van de cookies niet afhankelijk van wat een "gebruiker" van de cookie-consent ermee doet? is het dan OK om hier (in library code toch?) namen van cookies "vast" te leggen?

Reply:
Het is inderdaad niet de bedoeling op cookie te delen, als dat interessant zou zijn kan die verplaatst worden naar de vl-ui-core en moeten de vl-ui-cookie-consent specifieke cookies er natuurlijk uit verwijderd worden.

Misschien niet slecht om de cookie page objecten te verplaatsen. Lijkt me wel handig te zijn voor afnemers dat ze dit niet opnieuw zelf moeten implementeren.

Reply:
Extractie gemaakt en toegevoegd aan de core.